lib/src/components/NxTag/NxTag.scss

```
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ @import '../../scss-shared/nx-variables';
+ @import '../../scss-shared/nx-text-helpers';
+ @import '../../scss-shared/nx-container-helpers';
+
+.nx-tag {
+  @include semi-bold();
+  @include container-horizontal;
+  @include nx-truncate-ellipsis;
+  background-color: $nx-blue-800;
+  border: 1px solid $nx-blue-800;
+  border-radius: 16px;
+  box-sizing: border-box;
+  color: $nx-text-color-dark;
+  cursor: default;
+  display: inline-block;
+  font-size: $nx-font-size-s;
+  height: 32px;
+  margin: 0 $nx-spacing-xs $nx-spacing-xs 0;
+  padding: 5px $nx-spacing-md 6px $nx-spacing-md;
+  // Uppercase Min so SCSS ignores it and uses CSS min, not SCSS min
+  max-width: Min(100%, 320px);
+}
+
+.nx-tag--selectable {
+  align-items: center;
+  cursor: pointer;
+  display: inline-flex;
+  padding-right: $nx-spacing-xs;
+
+  &:focus {
+    border: 1px solid $nx-focus-border-color;
+    box-shadow: $nx-focus-box-shadow;
+    outline: 0;
+  }
+
+  &:hover {
+    border-color: $nx-grey-900;
+  }
+}
+
+.nx-tag--selected {
+  background-color: $nx-blue-800;
+  border-color: $nx-blue-800;
+  color: $nx-text-color-dark;
+
+  &:focus {
+    border: 1px solid $nx-focus-border-color;
+  }
+}
+
+.nx-tag--unselected {
+  border-color: $nx-grey-300;
+}
+
+.nx-tag__text {
+  @include nx-truncate-ellipsis;
+}
+
+.nx-tag__action {
+  margin-left: $nx-spacing-md;
+}
+
+// Available tag colours
+.nx-tag--default {
+  color: $nx-white;
+
+  &.nx-tag--unselected {
+    background-color: $nx-grey-300;
+    color: $nx-text-color-dark;
+  }
+}
+
+@mixin tag-color($tag-name, $tag-color-dark, $tag-color-light) {
+  .nx-tag--#{$tag-name} {
+    background-color: $tag-color-dark;
+    border-color: $tag-color-dark;
+  
+    &.nx-tag--unselected {
+      background-color: $tag-color-light;
+    }
+  }
+}
+
+@include tag-color(light-blue, $nx-blue-200, $nx-blue-300);
+
+@include tag-color(purple, $nx-purple-200,#f3dcff);
+
+@include tag-color(pink, $nx-pink-300, #ffdbf0);
+
+@include tag-color(blue, $nx-teal-400, #b5eaff);
+
+@include tag-color(red, $nx-red-300, #ffd7d8);
+
+@include tag-color(green, $nx-turquoise-600, #befffa);
+
+@include tag-color(orange, $nx-orange-700, #ffe4c7);
+
+@include tag-color(yellow, $nx-yellow-700, #ffeab7);
+
+@include tag-color(lime, $nx-green-50, #f2ffec);
+
+@include tag-color(indigo, #919fe8, #eef0ff);
```

No need for all these blank lines IMO.
```suggestion
@include tag-color(light-blue, $nx-blue-200, $nx-blue-300);
@include tag-color(purple, $nx-purple-200,#f3dcff);
@include tag-color(pink, $nx-pink-300, #ffdbf0);
@include tag-color(blue, $nx-teal-400, #b5eaff);
@include tag-color(red, $nx-red-300, #ffd7d8);
@include tag-color(green, $nx-turquoise-600, #befffa);
@include tag-color(orange, $nx-orange-700, #ffe4c7);
@include tag-color(yellow, $nx-yellow-700, #ffeab7);
@include tag-color(lime, $nx-green-50, #f2ffec);
@include tag-color(indigo, #919fe8, #eef0ff);
```